docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
I think we need to add a call to `install-yarn.sh` here.  

(As a sidenote, I think we don't need to list node as something to install via `apt/dnf` above, since the line above this installs it).